packages/backend-common/api-report.md

```diff
@@ -145,6 +147,29 @@ export interface ContainerRunner {
   runContainer(opts: RunContainerOptions): Promise<void>;
 }
 
+// @public
+export interface Context {
+  readonly abortSignal: AbortSignal_2;
+  readonly deadline: Date | undefined;
+  use(...decorators: ContextDecorator[]): Context;
```

Comment:
Thoughts on dropping this and merging as experimental? :grin:
Along with `setAbort(src)` -> `withAbort(ctx, src)` etc.

Reply:
I'd vote to leave them in, mostly based on wanting to start to strongly encourage their use all over the place, with a low discoverability / code size cost at the use site. Feels like the responsible thing to do :D And so easy to pass through anyway that I don't mind.

Reply:
Leave what in? I'm not suggesting removing anything else than `use()`, it's just that we'd need to tweak the static methods.

Reply:
Ah I thought you meant the two above it too, since I didn't see how the `set`/`with` thing might come into play otherwise. Hm yeah I could totally go for removing `use` for now but, then what did you mean by the "Along with" thing? Should we _add_ those setters on the `Context` interface?

Reply:
The current `Contexts.setAbort(src)(ctx)` is pretty awkward without `use()`, so I'm suggesting we make it `Contexts.withAbort(ctx, src)`, and same for the rest of the decorator creators.